### PR TITLE
Gus Redesign

### PR DESCRIPTION
**Pull request by l0k0ms**

### CODE REVIEW

File: content/agent/proxy.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+* `/opt/local/datadog/supervisord/supervisord.conf` on SmartOS
+* `/usr/local/etc/datadog/supervisord/supervisord.conf` on FreeBSD
+* `~/.datadog-agent/supervisord/supervisord.conf` on Mac OS X
+
+Assuming that file is found at `SUP_FILE`
+```bash
+sed -i 's/ddagent.py/ddagent.py --sslcheck=0/' SUP_FILE
+```
+
+#### On Windows (Starting from agent 3.9.2):
```
Comment (technovangelist) [inline content/agent/proxy.md]: how long do we need to mention when some feature started? 3.9.2 was a loooong time ago.

File: content/agent/proxy.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    ```
+
+3. Allow non-local traffic on `proxy-node` by changing the following line in `/etc/dd-agent/datadog.conf`.  
+     `# non_local_traffic: no` should read `non_local_traffic: yes`.
+
+4. Make sure `proxy-node` can be reached from the other nodes over port 17123. Start the agent on the `proxy-node` and run on the other nodes:
+
+    `curl -v http://proxy-node:17123/status 2>&1 | grep "200 OK"`
+
+5. Update non-proxy nodes to forward to `proxy-node`. Simply change the following line in `/etc/dd-agent/datadog.conf` from:
```
Comment (technovangelist) [inline content/agent/proxy.md]: we are listing a specific path a few times, but the path is different (according to our docs for at least 2 platforms (mac and windows)), should we rephrase to the datadog.conf file (look in the agent docs for the location on your platform)?

File: content/agent/proxy.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+```
+
+#### On Windows (Starting from agent 3.9.2):
+
+Edit your configuration file (datadog.conf) and add this option:
+```
+skip_ssl_validation: yes
+```
+
+Finally [restart the agent](/agent/faq/start-stop-restart-the-datadog-agent).
```
Comment (technovangelist) [inline content/agent/proxy.md]: should the link be /agent/faq/start-stop-restart-the-datadog-agent/#windows

File: content/agent/proxy.md (hunk 2 above)
Comment (l0k0ms) [reply content/agent/proxy.md]: :+1:

File: content/agent/proxy.md (hunk 1 above)
Comment (l0k0ms) [reply content/agent/proxy.md]: I don't know :/ We should decide a DD agent minimum version supported on our doc.

File: content/agent/proxy.md (hunk 3 above)
Comment (l0k0ms) [reply content/agent/proxy.md]: 👍

File: content/graphing/miscellaneous/functions.md [hunk 4]
```diff
@@ -224,6 +224,8 @@ The method can be sum/min/max/count/avg and time is in seconds. You can use eith
 
 These functions are only intended for metrics submitted as rates or counters via statsd. These functions will have no effect for other metric types. For more on details about how to use `.as_count()` and `.as_rate()` please see [our blog post][1].
 
+Note: The only available query with `as_count()` is `sum()` (unless using a rollup summary), which is the only mathematical accurate function with such behavior.
```
Comment (technovangelist) [inline content/graphing/miscellaneous/functions.md]: so if you use rollup summary, what other query is available? and what does 'which' refer to? re: the only math accurate function. And what does 'with such behavior' refer to?

File: content/developers/dogstatsd.md [hunk 5]
```diff
@@ -50,9 +50,17 @@ where it will be stored and available for graphing alongside the rest of your me
 
 ## Setup
 
-Once you have the Datadog Agent installed on your application servers/containers—or anywhere
-your application can reliably reach—grab the [DogStatsD client library][2] for your
-application language and you'll be ready to start hacking. You _can_ use any generic StatsD client to send metrics to DogStatsD, but you won't be able to use any of the Datadog-specific features mentioned above.
+First, take a look at your `datadog.conf` file, and uncommented the following lines:
```
Comment (technovangelist) [inline content/developers/dogstatsd.md]: do you just want to look at the file? First, edit your `datadog.conf` file to uncomment the following lines:

File: content/developers/dogstatsd.md [hunk 6]
```diff
@@ -50,9 +50,17 @@ where it will be stored and available for graphing alongside the rest of your me
...
+``` 
+# If you don't want to enable the DogStatsd server, set this option to no
```
Comment (technovangelist) [inline content/developers/dogstatsd.md]: wait, uncomment this? If not, don't include this line here

File: content/developers/dogstatsd.md [hunk 7]
```diff
@@ -50,9 +50,17 @@ where it will be stored and available for graphing alongside the rest of your me
...
+use_dogstatsd: yes
```
Comment (technovangelist) [inline content/developers/dogstatsd.md]: isn't this the default? so why uncomment?

File: content/developers/dogstatsd.md [hunk 8]
```diff
@@ -50,9 +50,17 @@ where it will be stored and available for graphing alongside the rest of your me
...
+# Make sure your client is sending to the same port.
+ dogstatsd_port: 8125
```
Comment (technovangelist) [inline content/developers/dogstatsd.md]: this isn't the default?

File: content/developers/dogstatsd.md [hunk 9]
```diff
@@ -50,9 +50,17 @@ where it will be stored and available for graphing alongside the rest of your me
...
+```
+
+Then [restart your agent](/agent/faq/start-stop-restart-the-datadog-agent).
+
+Once done, your application can reliably reach—grab the [DogStatsD client library][2] for your application language and you'll be ready to start hacking. You _can_ use any generic StatsD client to send metrics to DogStatsD, but you won't be able to use any of the Datadog-specific features mentioned above.
```
Comment (technovangelist) [inline content/developers/dogstatsd.md]: what does reach-grab mean?

File: content/developers/dogstatsd.md (hunk 6 above)
Comment (l0k0ms) [reply content/developers/dogstatsd.md]: 👍

File: content/developers/dogstatsd.md (hunk 5 above)
Comment (l0k0ms) [reply content/developers/dogstatsd.md]: 👍

File: content/developers/dogstatsd.md (hunk 8 above)
Comment (l0k0ms) [reply content/developers/dogstatsd.md]: depend of the agent version

File: content/developers/dogstatsd.md (hunk 9 above)
Comment (l0k0ms) [reply content/developers/dogstatsd.md]: 👍

**Pull request merged**